src/Profile/Rule.cs

```diff
@@ -242,7 +243,8 @@ static double Variance(ProtoCrewMember c, double variance)
 		public double variance;           // variance for degeneration rate, unique per-kerbal and in range [1.0-variance, 1.0+variance]
 		public List<string> modifiers;    // if specified, rates are influenced by the product of all environment modifiers
 		public bool breakdown;            // if true, trigger a breakdown instead of killing the kerbal
-		public bool output_only;          // if true and input resource exists only monitor the input resource, do not consume it
+		public bool monitor;              // if true and input resource exists only monitor the input resource, do not consume it
+		public double monitor_offset;     // add this percentage to input resource to stop output and alter trigger [range 0 to 1]
```

Comment:
@PiezPiedPy If i get this correctly output stops if input reaches a certain condition, can you clarify this in the comment and choose a more descriptive name if possible?